internal/db/db.go

```diff
@@ -163,7 +165,7 @@ func InitDBConnectionPool(ctx context.Context, connStr string) (DB, error) {
 		return nil, err
 	}
 
-	dbConfig.ConnConfig.Tracer = &dbTracer{}
+	dbConfig.ConnConfig.Tracer = sinit.PgxTracer(slog.Default())
```

Comment:
The `dbTracer` utility type and the test were moved into `logging` library. See here:

https://github.com/osbuild/logging/pull/20